src/borg/helpers/parseformat.py

```diff
@@ -648,6 +648,9 @@ def format_time(self, ts):
 
 
 class ItemFormatter(BaseFormatter):
+    from .algorithms.checksums import StreamingXXH64
```

Comment:
hmm, this worked, but putting the same line in global module scope did not?

Reply:
Yes, this works and neither that same import nor any other variant I came up with did, including a plain `import` without `from` - guess it would have to be that in both modules to make cyclic import work, or maybe Cython is a special case here...

Reply:
Correction, this does work on 1.1, , not with 1.2 though :thinking: